projects/storefrontstyles/scss/components/cart/_cart-details.scss

```diff
@@ -17,4 +17,15 @@
     color: var(--cx-color-success);
     padding: 0.5rem 0;
   }
+
+  .cx-remove-btn {
+    text-align: right;
```

Comment:
Although this is not a breaking change in itself for this scss file... **it is for the HTML** where you're using a Feature Flag, plus, unfortunately we already have a **calydon** instance for this file in this location:
`projects/storefrontstyles/scss/themes/calydon/components/cart/_cart-details.scss`
therefore, for consistency, please transfer all **new** styles to the **calydon** instance

Reply:
Hi @developpeurweb could you share more detail about the calydon style? I checked the projects/storefrontstyles/scss/themes/calydon/components/cart/_cart-details.scss and still not sure how to update this one.

Reply:
@maggielin0 pls follow this procedure:
1. cut/paste lines 20 to 31 from original `_cart-details.scss` into `calydon/components/cart/_cart-details.scss`
2. In order to see your changes you'll have to uncomment line 4 in `projects/storefrontapp/src/styles.scss` **(do not commit this file)**.

By default **Calydon** instance is OFF, and it's opt-in for the client (like the feature flag).

Keep in mind all style additions to existing components should be done in the calydon instance to avoid breaking changes, if the calydon file already exist, feel free to add and adjust. If file doesn't exist in calydon instance pls copy the original structure and use existing calydon structure as reference, if in doubt, please contact me.

Reply:
Thanks for this elaborate instruction. Updated the style.